interface/forms/vitals/growthchart/chart.php

```diff
@@ -258,8 +258,8 @@ function cssHeader()
     ?>
     <html>
     <head>
-    <link rel="stylesheet" type="text/css" title="page1" href="page1.css">
-    <link rel="stylesheet" type="text/css" title="page2" href="page2.css">
+    <link rel="stylesheet" title="page1" href="page1.css">
+    <link rel="stylesheet" title="page2" href="page2.css">
```

Comment:
@tywrenn 

Should I remove `title="page1"`, Is it necessary?

Reply:
Check to make sure it isn't triggered by JavaScript or jquery and if it isn't it should be removed.

Reply:
I found this attribute can implement a change page style feature. So l leave those attributes.

https://developer.mozilla.org/en-US/docs/Web/CSS/Alternative_style_sheets